CONTRIBUTING.md

```diff
@@ -1,173 +1,95 @@
 <!--- # NOTE: Modify sections marked with `TODO` and then rename the file.-->
 
-# How to Contribute
+# How to Contribute (Internal)
 
-We're so thankful you're considering contributing to an [open source project of
-the U.S. government](https://code.gov/)! If you're unsure about anything, just
-ask -- or submit the issue or pull request anyway. The worst that can happen is
-you'll be politely asked to change something. We appreciate all friendly
-contributions.
+The following guide is for members of the project team who have access to the repository. For external contributors, see: [CONTRIBUTING_EXTERNAL.md](./CONTRIBUTING_EXTERNAL.md).
 
-We encourage you to read this project's CONTRIBUTING policy (you are here), its
-[LICENSE](LICENSE.md), and its [README](README.md).
+## Local Development
 
-## Getting Started
-<!--- ### TODO: If you have 'good-first-issue' or 'easy' labels for newcomers, mention them here.-->
+This project is monorepo with several apps. Please see the [api](./api/README.md) and [frontend](./frontend/README.md) READMEs for information on spinning up those projects locally. Also see the project [documentation](./documentation) for more info.
 
-### Team Specific Guidelines
+### Linting and Testing
 
-### Building dependencies
+Each application has its own linting and testing guidelines. Lint and code tests are run on each commit, so linters and tests should be run locally before commiting.
 
-<!--- ### TODO -->
+## Branching Model
 
-### Building the Project
+This project follows [trunk-based development](https://trunkbaseddevelopment.com/), which means:
 
-<!--- ### TODO -->
+* Make small changes in [short-lived feature branches](https://trunkbaseddevelopment.com/short-lived-feature-branches/) and merge to `main` frequently.
+* Be open to submitting multiple small pull requests for a single ticket (i.e. reference the same ticket across multiple pull requests).
+* Treat each change you merge to `main` as immediately deployable to production. Do not merge changes that depend on subsequent changes you plan to make, even if you plan to make those changes shortly.
+* Ticket any unfinished or partially finished work.
+* Tests should be written for any change introduced.
```

Comment:
Maybe add that test coverage should not dip below 80% or something like that since we have a threshold set up for the backend.